05.wc/wc.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+require 'optparse'
+
+def parse_options
+  options = {}
+  OptionParser.new do |opts|
+    opts.on('-l') { options[:lines] = true }
+    opts.on('-w') { options[:words] = true }
+    opts.on('-c') { options[:bytes] = true }
+  end.parse!
+  options = { bytes: true, lines: true, words: true } if options.empty?
+  [options, ARGV.empty? ? [ARGF] : ARGV]
```

Comment:
```suggestion
  hoge = ARGV.empty? ? [ARGF] : ARGV
  [options, hoge]
```

配列リテラルの中に三項演算子が入り、さらにその中に別の配列リテラルが入るというのはちょっと可読性が悪いです。
一気にやろうとせず変数に入れましょう。

https://x.com/jnchito/status/1326468865103073280